deps/chakrashim/src/inspector/v8-inspector-session-impl.cc

```diff
@@ -28,7 +31,9 @@ bool V8InspectorSession::canDispatchMethod(const StringView& method) {
          stringViewStartsWith(method,
                               protocol::Console::Metainfo::commandPrefix) ||
          stringViewStartsWith(method,
-                              protocol::Schema::Metainfo::commandPrefix);
+                              protocol::Schema::Metainfo::commandPrefix) ||
+         stringViewStartsWith(method,
```

Comment:
There are currently no usages of ENABLE_TTD_NODE anywhere in chakrashim.  I even see one instance where it's present, but commented out, so I suspect it's not possible.